llvm-project/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+wget --no-clobber https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.1/llvm-project-14.0.1.src.tar.xz
+docker build --tag clangbuiltlinux/containers:llvm-project . -f Dockerfile.bootstrap
+#docker create --name llvm-project clangbuiltlinux/containers:llvm-project
```

Comment:
These command I'm not super familiar with.  Should I be putting a 14.0.1 version in there somewhere, or somehow denoting this is a bootstrap version?

Not sure how to publish this to our container registry either, but that should be start for fetching the sysroot within in.

Reply:
Also, if I comment back in the commented out line and rerun locally, I start to get errors. Not sure yet how best to handle that.

Reply:
> Should I be putting a 14.0.1 version in there somewhere, or somehow denoting this is a bootstrap version?

It certainly does not hurt.

The tag argument should be

`--tag clangbuiltlinux/llvm-project:latest`

as `clangbuiltlinux` is the user and `llvm-project` is the image name:

https://github.com/orgs/ClangBuiltLinux/packages

You can specify multiple tags, so you could do:

`--tag clangbuiltlinux/llvm-project:latest --tag clangbuiltlinux/llvm-project:14.0.1`

or whatever.

Adding `ghcr.io/` before all the tag names should make it easy to push to ghcr.io with [`docker login` and `docker push`](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or we can just wire up the existing CI to do it all for us.

I am not sure why the `create` command would fail... perhaps lack of a `CMD`?

https://docs.docker.com/engine/reference/builder/#cmd

Reply:
SGTM; ideally no one should be able to push the containers, I think, just the CI. Done, PTAL